src/test/java/com/coveo/pushapiclient/PlatformClientTest.java

```diff
@@ -124,6 +124,24 @@ public void testCreateSource() throws IOException, InterruptedException {
         assertEquals(true, requestBody.get("pushEnabled"));
     }
 
+    @Test
```

Comment:
I feel like this function is testing everything (source visibility, name, source type, ...)
It can be split into multiple smaller tests